core/jobs.py

```diff
@@ -579,7 +582,10 @@ def _real_enqueue(cls, job_id, queue_name, additional_job_params):
 
 
 class MapReduceJobPipeline(base_handler.PipelineBase):
-
+    """MapreducePipeline class which will be used later to call map
+    and reduce calculations. It implements run method which is called
+    by inherited start method of this class.
```

Comment:
This is a bit opaque. Can you write it in plain English? "... which is called when XXX happens." perhaps?

Reply:
updated.